automl/cloud-client/get_operation_status_test.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from google.cloud import automl
+import pytest
+
+import get_operation_status
+
+PROJECT_ID = os.environ["AUTOML_PROJECT_ID"]
+
+
+@pytest.fixture(scope="function")
+def get_operation_id():
+    client = automl.AutoMlClient()
+    project_location = client.location_path(PROJECT_ID, "us-central1")
+    response = client.transport._operations_client.list_operations(
```

Comment:
This looks like it's doing something wrong - why is this calling an underscored method? Does the higher level client not provide a function for the endpoint?

Reply:
The python client library doesn't seem to give you direct access to the operations client, while Java and Node.js do:

Java:
`client.getOperationsClient()`
Node.js
`client.operationsClient`

Reply:
nit: Did we file a bug against the client library for this?

Reply:
Oh, yes, filed now.